lib/docblockr-worker.js

```diff
@@ -1,14 +1,17 @@
+/* eslint indent: ['error', 2], no-redeclare: 'off', no-undef: 'off' */
```

Comment:
> Probably you should clean up docblockr-worker.js and other files to remove some turned off eslint rules.

I mostly where referring to this line, this whole file has an indentation of 2 spaces instead of 4. And some extra exception to make it validate against eslint.

Reply:
Yeah but never the less there would be *a lot* more to clean up